kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -163,12 +163,20 @@ def before_command_run(
 
             consent = _check_for_telemetry_consent(project_metadata.project_path)
             if not consent:
-                logger.debug(
+                logger.info(
                     "Kedro-Telemetry is installed, but you have opted out of "
                     "sharing usage analytics so none will be collected.",
                 )
                 return
 
+            logger.info(
+                "Kedro is sending anonymous usage data with the sole purpose of improving the product. "
+                "No personal data or IP addresses are stored on our side. "
+                "If you want to opt out, set the `KEDRO_DISABLE_TELEMETRY` or `DO_NOT_TRACK` environment variables, "
+                "or create a `.telemetry` file in the current working directory with the contents `consent: false`. "
+                "Read more at https://docs.kedro.org/en/latest/configuration/telemetry.html"
```

Comment:
Should we point the docs to `stable` instead? That's the most recent released version. `latest` is equivalent to non-released content.

Reply:
Thanks, fixed